packages/mobile/src/screens/profile-screen/ProfileInfo.tsx

```diff
@@ -86,7 +86,8 @@ export const ProfileInfo = (props: ProfileInfoProps) => {
     'name',
     'handle',
     'does_current_user_follow',
-    'does_follow_current_user'
+    'does_follow_current_user',
+    'is_verified'
```

Comment:
oh dang, this is kind of scary because nowhere in this file do we reference is_verified explicitly. wondering if we can be more deliberate than to pass `profile` down all the way through the component tree?

Reply:
yeah for sure. i just typed the `useSelectProfile` hook to be correct because before it coerced the return type to be the full User object. this would have prevented this bug

Reply:
agree its more reacty and better to pass the individual values of `profile` that we need in the presentational components, but a bunch of components (including two more in this same file) follow the same pattern of passing in a whole `profile` object so I just left that for now

Reply:
yeah this is super scary, and my bad honestly, what should we do?

Reply:
the manual typing is good. probably stop passing objects through is best or selecting from redux if you need a complex object and not depending on parent to pass it